docs/content/commands/npm-update.md

```diff
@@ -101,6 +103,36 @@ Then `npm update` will install `dep1@0.4.1`, because that is the highest-sorting
 version that satisfies `^0.4.0` (`>= 0.4.0 <0.5.0`)
 
 
+#### Subdependencies
+
+Suppose your app now also has a dependency on `dep2`
+
+```json
+{
```

Comment:
I think adding something like `"name": "my-app"` would make this example easier to understand at first glance, even though the same info is there in the preceding sentence.